server/src/RescriptEditorSupport.ts

```diff
@@ -38,16 +38,16 @@ export function runDumpCommand(msg: RequestMessage): dumpCommandResult | null {
   }
 
   let command =
-    executable.binaryPathQuoted +
-    " dump " +
     executable.filePathQuoted +
```

Comment:
The argument should not be quoted anymore now that an array of arguments is passed.

Reply:
I'm modifying the binary right now. `execFileSync`'s arguments array's each item is considered as a space-separated command. Currently this still needs quoting since we concatenate the file path with the 2 position arguments. If we want to leverage proper escaping from `execFileSync`, we need to stop concatenating too. Aka the binary's position arguments shouldn't be passed as `filePath:line:col` but `filePath line col`.

Reply:
Nvm, you're right. This... works. We'll revisit the binary's `path:line:col` argument parsing later. Less parsing = less error